wdl/Utils.wdl

```diff
@@ -44,6 +44,138 @@ task GetSampleIdsFromVcf {
+      bcftools query -l $VCF >> ~{prefix}.txt
+    done < ~{write_lines(vcfs)}
+
+  >>>
+
+  output {
+    File out_file = "~{prefix}.txt"
+    Array[String] out_array = read_lines("~{prefix}.txt")
+  }
+
+  runtime {
+    cpu: select_first([runtime_attr.cpu_cores, default_attr.cpu_cores])
+    memory: select_first([runtime_attr.mem_gb, default_attr.mem_gb]) + " GiB"
+    disks: "local-disk " + select_first([runtime_attr.disk_gb, default_attr.disk_gb]) + " HDD"
+    bootDiskSizeGb: select_first([runtime_attr.boot_disk_gb, default_attr.boot_disk_gb])
+    docker: sv_base_mini_docker
+    preemptible: select_first([runtime_attr.preemptible_tries, default_attr.preemptible_tries])
+    maxRetries: select_first([runtime_attr.max_retries, default_attr.max_retries])
+  }
+}
+
+task GetSampleIdsFromVcfTar {
+  input {
+    File vcf_tar
+    String prefix
+    String sv_base_mini_docker
+    RuntimeAttr? runtime_attr_override
+  }
+
+  RuntimeAttr default_attr = object {
+                               cpu_cores: 1,
+                               mem_gb: 0.9,
+                               disk_gb: 10 + 2 * ceil(size(vcf_tar, "GiB")),
+                               boot_disk_gb: 10,
+                               preemptible_tries: 3,
+                               max_retries: 1
+                             }
+  RuntimeAttr runtime_attr = select_first([runtime_attr_override, default_attr])
+
+  command <<<
+
+    set -eu
+    mkdir vcfs
+    tar xzf ~{vcf_tar} -C vcfs/
+    for VCF in vcfs/*.vcf.gz; do
+      bcftools query -l $VCF
+    done > ~{prefix}.txt
+
+  >>>
+
+  output {
+    File out_file = "~{prefix}.txt"
+    Array[String] out_array = read_lines("~{prefix}.txt")
+  }
+
+  runtime {
+    cpu: select_first([runtime_attr.cpu_cores, default_attr.cpu_cores])
+    memory: select_first([runtime_attr.mem_gb, default_attr.mem_gb]) + " GiB"
+    disks: "local-disk " + select_first([runtime_attr.disk_gb, default_attr.disk_gb]) + " HDD"
+    bootDiskSizeGb: select_first([runtime_attr.boot_disk_gb, default_attr.boot_disk_gb])
+    docker: sv_base_mini_docker
+    preemptible: select_first([runtime_attr.preemptible_tries, default_attr.preemptible_tries])
+    maxRetries: select_first([runtime_attr.max_retries, default_attr.max_retries])
+  }
+}
+
+task GetSampleIdsFromVcfList {
+  input {
+    File vcf_list
+    String prefix
+    String sv_base_docker
+    RuntimeAttr? runtime_attr_override
+  }
+
+  RuntimeAttr default_attr = object {
+                               cpu_cores: 1,
+                               mem_gb: 0.9,
+                               disk_gb: 100,
+                               boot_disk_gb: 10,
+                               preemptible_tries: 3,
+                               max_retries: 1
+                             }
+  RuntimeAttr runtime_attr = select_first([runtime_attr_override, default_attr])
+
+  command <<<
+    set -euo pipefail
+    mkdir vcfs
+    cat ~{vcf_list} | gsutil -m cp -I vcfs/
```

Comment:
Seems like you could avoid downloading all the files. If you are GCS authed and our version of bcftools is modern enough you should be able to `bcftools query -l` the remote files one by one. Or you could `gsutil cat` them and pipe it to `bcftools query -l` although you have to swallow a broken pipe error there.

Reply:
Actually it looks like this isn't used anymore so I've deleted the task